source/mir/ndslice/slice.d

```diff
@@ -1007,22 +1018,26 @@ public:
     {
         private alias ConstThis = Slice!(const(Unqual!(PointerTarget!Iterator))*, N, kind);
         private alias ImmutableThis = Slice!(immutable(Unqual!(PointerTarget!Iterator))*, N, kind);
+        private alias ImmutableOfUnqualOfPointerTarget(Iterator) = immutable(Unqual!(PointerTarget!Iterator))*;
+        private alias ConstOfUnqualOfPointerTarget(Iterator) = const(Unqual!(PointerTarget!Iterator))*;
 
         /++
         Cast to const and immutable slices in case of underlying range is a pointer.
         +/
         auto toImmutable()() scope return immutable @trusted pure nothrow @nogc
         {
             alias It = immutable(Unqual!(PointerTarget!Iterator))*;
```

Comment:
can be simplified using ImmutableOfUnqualOfPointerTarget

Reply:
Addressed in d20ea47